legislation_analysis/api/congress.py

```diff
@@ -44,6 +45,25 @@ def __init__(self, file_path):
         self.processed_df = None
         self.df = None
 
+    def get_df(self) -> None:
```

Comment:
I'd change the name of the function to `set` instead of `get`. `get` infers that you are going to return that object.